src/MadeTech/CacheWarming/Gateway/FileGetContentsUrlRetriever.php

```diff
@@ -7,6 +7,6 @@ class FileGetContentsUrlRetriever implements UrlRetriever
 {
     public function get($url)
     {
-        return file_get_contents($url);
+        return @file_get_contents($url);
```

Comment:
See ```FileGetContentsUrlRetrieverTest``` we want timeouts/failures to silently fail and cause the application continue to crawl other URLs.

There's a test in place which asserts this is true, but it spams the console during testing